apps/aecore/src/aec_object_serialization.erl

```diff
@@ -70,6 +70,15 @@ tag(contract) -> 40;
 tag(contract_call) -> 41;
 tag(contract_create_tx) -> 42;
 tag(contract_call_tx) -> 43;
+tag(channel_create_tx) -> 50;
```

Comment:
Could you please propose PR in protocol repo with serialization update? https://github.com/aeternity/protocol/blob/80be1c7ea399770367635479cc32fb76482f7646/serializations.md

Reply:
[protocol PR 74](https://github.com/aeternity/protocol/pull/74) created